src/test/java/org/sonarsource/scanner/maven/SonarQubeMojoTest.java

```diff
@@ -104,6 +105,50 @@ public void shouldExportOverridenWarWebSource() throws Exception {
         + new File(baseDir, "src/main/java").getAbsolutePath()));
   }
 
+  @Test
+  public void project_with_java_files_not_in_src() throws Exception {
```

Comment:
Can we name the test in a way that expresses what we want to assert?

Reply:
fixed